### PR TITLE
murmur3 should output 128 bit instead of 32

### PR DESCRIPTION
**Pull request by pgte**
...to be compatible with IPFS-Go's implementation

### CODE REVIEW

**Comment by dignifiedquire** [comment]
@pgte this doesn't look correct me, go-multihash doesn't use 128bit: https://github.com/multiformats/go-multihash/blob/master/sum_test.go#L30

**Comment by pgte** [comment]
@dignifiedquire it uses 64 bit, but murmur3 doesn't have a 64 bit version. What go does is produce a 128 bit hash and only use the 64 bits. (in JS, unixfs-engine is doing the trimming).

**Comment by dignifiedquire** [comment]
this sounds fishy @Kubuxu @whyrusleeping could you shed some light on this please

**Comment by diasdavid** [comment]
@dignifiedquire, @pgte is correct, @whyrusleeping and Pedro are up to sync on this.

**Comment by dignifiedquire** [comment]
So go-multihash will upgrade as well?

**Comment by diasdavid** [comment]
The codec needs to change to `murmur3-128`

**Comment by pgte** [comment]
Since the name changed, needs a new release of the js-multihashes package.

File: benchmarks/hash.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const algs = [
   'keccak-256',
   'keccak-384',
   'keccak-512',
-  'murmur3'
+  'murmur3-128'
```
Comment (dignifiedquire) [inline benchmarks/hash.js]: Lets add the short one as well while we are at it

Comment (pgte) [inline benchmarks/hash.js]: done

**Pull request merged**